### PR TITLE
add the ability to retrieve data from the stream

### PR DESCRIPTION
**Pull request by Scandinaf**

### CODE REVIEW

File: scassandra/src/main/scala/com/evolutiongaming/scassandra/ResultSet.scala [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+import cats.effect.{Concurrent, Sync}
+import cats.effect.implicits._
+import cats.syntax.all._
+import com.datastax.driver.core.{Row, ResultSet => ResultSetJ}
+import com.evolutiongaming.scassandra.util.FromGFuture
+import com.evolutiongaming.sstream.Stream
+import com.evolutiongaming.sstream.FoldWhile._
+
+
+object ResultSet {
```
Comment (t3hnar) [inline scassandra/src/main/scala/com/evolutiongaming/scassandra/ResultSet.scala:12]: in order to not conflict with `ResultSet` class of cassandra, could we rather add this is implicit extension method to `ResultSet` in `syntax.scala` ?

Comment (Scandinaf) [reply scassandra/src/main/scala/com/evolutiongaming/scassandra/ResultSet.scala:12]: Great idea, let's transfer this extension to `scassandra.syntax`.

Comment (Scandinaf) [reply scassandra/src/main/scala/com/evolutiongaming/scassandra/ResultSet.scala]: @t3hnar Done.

**Comment by t3hnar** [comment]
Thanks a lot @Scandinaf

**Pull request merged**